app/views/api/v1/general_informations/show.json.jbuilder

```diff
@@ -0,0 +1,4 @@
+json.general_information do
+  json.title    @info.title
```

Comment:
It'd be better if this view calls a partial `_info` so this partial can be reused